src/Sentry/SentryStructuredLogger.Format.cs

```diff
@@ -0,0 +1,156 @@
+using Sentry.Infrastructure;
+
+namespace Sentry;
+
+public abstract partial class SentryStructuredLogger
```

Comment:
note: made class `partial` and moved all overloads of the `Log{Level}` method groups to a separate document